### PR TITLE
translations

### PR DESCRIPTION
**Pull request by deepak2510**

### CODE REVIEW

File: react_components/Card.js [hunk 1]
```diff
@@ -133,19 +156,21 @@ class CardComponent extends React.Component {
...
+                {formatMessage(messages.deleteBioText)}
+            </button>
             <br /><br />
             <Alert text={this.state.alerts} />
 
             <form>
               <div className="input-group cardComponent">
                 <div className="input-group-btn cardComponent">
-                  <button ref="likeButton" className="btn btn-default cardComponent" onClick={this.like}>+1</button>
+                  <button ref="likeButton" className="btn btn-default cardComponent likeBut" onClick={this.like}>+1</button>
```
Comment (aaronkaka) [inline react_components/Card.js]: Remove class "likeBut"... looks like a typo.

File: react_components/Card.js [hunk 2]
```diff
@@ -118,10 +134,17 @@ class CardComponent extends React.Component {
 
   render() {
 
+    const {formatMessage} = this.props.intl;
     const data = this.state.data;
 
     return (
       <div className="panel panel-default cardComponent">
+          <FormattedMessage
```
Comment (aaronkaka) [inline react_components/Card.js]: please remove this

File: __tests__/card.js [hunk 3]
```diff
@@ -6,39 +6,67 @@ import React from 'react';
...
 
-  const CardElement = TestUtils.renderIntoDocument(
-    <Card data={targetData} />
-  );
+    renderer.render(
+          <Card.WrappedComponent
+              data={targetData}
+              intl = {intl}
+              />
+    );
```
Comment (aaronkaka) [inline __tests__/card.js]: remove spaces before and after =

File: react_components/Card.js (hunk 1 above)
Comment (aaronkaka) [inline react_components/Card.js]: change the name of the "likeBut" class to "shallowRenderRef"

File: __tests__/card.js [hunk 4]
```diff
@@ -6,39 +6,67 @@ import React from 'react';
...
+    expect(label.props.children.length).toEqual(0);
   });
 
   it('increments and displays the like count', function () {
-    const likeButton = ReactDOM.findDOMNode(CardElement.refs.likeButton);
-    TestUtils.Simulate.click(likeButton);
-    expect(ReactDOM.findDOMNode(label).textContent).toEqual('+1');
+    const likeButton = findWithClass(renderedTree, 'likeBut');
+    likeButton.props.onClick({ preventDefault: () => {} });
+    const renderedTree2 = renderer.getRenderOutput();
```
Comment (aaronkaka) [inline __tests__/card.js]: change the name of the "likeBut" class to "shallowRenderRef"

File: react_components/Card.js [hunk 5]
```diff
@@ -167,4 +193,4 @@ CardComponent.propTypes = {
   })
 };
 
-export default CardComponent;
+export default injectIntl(CardComponent);
```
Comment (aaronkaka) [inline react_components/Card.js]: add code comment that this injects this.props.intl which is used above to get the formatMessage method

File: react_components/index.js [hunk 6]
```diff
@@ -5,6 +5,16 @@ import '../css/fontello.css';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import CardComponent from './Card';
+import {addLocaleData, IntlProvider} from 'react-intl';
```
Comment (aaronkaka) [inline react_components/index.js]: remove addLocaleData

File: react_components/index.js [hunk 7]
```diff
@@ -5,6 +5,16 @@ import '../css/fontello.css';
...
+import frLocaleData from 'react-intl/lib/locale-data/fr'
```
Comment (aaronkaka) [inline react_components/index.js]: remove this line

File: react_components/index.js [hunk 8]
```diff
@@ -5,6 +5,16 @@ import '../css/fontello.css';
...
+import enLocaleData from 'react-intl/lib/locale-data/en'
```
Comment (aaronkaka) [inline react_components/index.js]: remove this line

File: react_components/index.js [hunk 9]
```diff
@@ -5,6 +5,16 @@ import '../css/fontello.css';
...
+
+addLocaleData(enLocaleData);
```
Comment (aaronkaka) [inline react_components/index.js]: remove this line

File: react_components/index.js [hunk 10]
```diff
@@ -5,6 +5,16 @@ import '../css/fontello.css';
...
+addLocaleData(frLocaleData);
```
Comment (aaronkaka) [inline react_components/index.js]: remove this line

File: react_components/index.js [hunk 11]
```diff
@@ -25,8 +35,18 @@ export default function cardstrap(containerType) {
         console.info(`New Event ${e.type} for ${targetElement}`);
         console.info(targetData);
 
+        const messages = {
```
Comment (aaronkaka) [inline react_components/index.js]: rename this to translations and move its definition out to right after the const frJson declaration

File: react_components/index.js [hunk 12]
```diff
@@ -25,8 +35,18 @@ export default function cardstrap(containerType) {
...
+          'en-US' : enJson,
+          'fr' : frJson
+        };
+
+        const locale = 'fr';
+
         ReactDOM.render(
-          <CardComponent data={targetData} key={keyId} />, document.getElementById(targetElement)
+            <IntlProvider locale={locale} messages={messages[locale]}>
```
Comment (aaronkaka) [inline react_components/index.js]: after the rename, would be translations[locale]

File: __tests__/card.js [hunk 13]
```diff
@@ -6,39 +6,67 @@ import React from 'react';
...
+import {IntlProvider} from 'react-intl';
+import {findWithType, findWithClass, findAllWithClass} from 'react-shallow-testutils';
+
+
+//const enJson = require('json!translations/en-US.json');
 
 describe('Card', function() {
+    const renderer = TestUtils.createRenderer();
+
+    console.log("RENDERER", renderer);
```
Comment (aaronkaka) [inline __tests__/card.js]: remove

File: react_components/Card.js [hunk 14]
```diff
@@ -3,6 +3,22 @@ import Avatar from './Avatar';
 import Bio from './Bio';
 import Alert from './Alert';
 
+import {FormattedMessage, intlShape, injectIntl, defineMessages} from 'react-intl';
```
Comment (aaronkaka) [inline react_components/Card.js]: remove FormattedMessage

File: react_components/index.js (hunk 6 above)
Comment (deepak2510) [inline react_components/index.js]: without importing that, how will I be able to add the specific locale data pertaining to specific languages ?

**Pull request merged**